link_scripts

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
```

Comment:
Maybe create a `bin/` directory to store this script (and potentially others in the future).